File: src/main/java/br/com/me/backendchallenge/domain/StatusPedido.java
package br.com.me.backendchallenge.domain;

import br.com.me.backendchallenge.enums.Status;
import com.fasterxml.jackson.annotation.JsonIgnore;
import lombok.Getter;
import lombok.Setter;

import javax.persistence.*;
import java.math.BigDecimal;

@Getter
@Setter
@Entity
@Table(name = "TB_STATUS_PEDIDO")
public class StatusPedido {
    @Id
    @Column(name = "ID")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @JsonIgnore
    @OneToOne
    private Pedido pedido;

    private Status status;
    private Long itensAprovados;
    private BigDecimal valorAprovado;
}

File: src/UI/Api/Models/Request/StatusPedido.cs
namespace Api.Models.Request
{
    public class StatusPedido
    {
        public string Status { get; set; }
        public int ItensAprovados { get; set; }
        public double ValorAprovado { get; set; }
        public string Pedido { get; set; }

    }
}